packages/@react-aria/slider/src/useSliderThumb.ts

```diff
@@ -121,7 +124,12 @@ export function useSliderThumb(
           setThumbValue(index, getThumbMaxValue(index));
           break;
       }
+
       setThumbDragging(index, false);
+      // Perform scroll after the thumb is fully positioned
+      requestAnimationFrame(() => {
+        scrollIntoViewFully(thumbRef.current, {block: 'center', inline: 'center'});
```

Comment:
the input of the slider thumb is actually visually hidden, so the IntersectionObserver will always think it isn't in the viewport. Added the `thumbRef` to get around this.

Also changed the scroll strategy to be  `block: 'center', inline: 'center'` which moves it to the center of the screen when using the end/page up/ etc to move the slider. I felt that using `block: 'nearest'` didn't really bring the slider into view enough and it may not be clear to the user that they just moved the slider thumb. Open to opinions here, try comparing the scrolling that happens when using the arrow keys to adjust the slider vs using page up/down/home/end

Reply:
Also we have other components that will scroll into view on keyboard nav (e.g. ActionGroup) via `.focus` which scrolls the newly focused element to the center of the screen. Might be nice to unify this behavior?